usr/local/www/services_ntpd.php

```diff
@@ -148,12 +148,14 @@
 		elseif (isset($config['ntpd']['notrap']))
 			unset($config['ntpd']['notrap']);
 
-		if ((empty($_POST['statsgraph'])) != (isset($config['ntpd']['statsgraph'])));
-			enable_rrd_graphing();
+		if ((empty($_POST['statsgraph'])) == (isset($config['ntpd']['statsgraph'])))
+			$enable_rrd_graphing = true;
 		if (!empty($_POST['statsgraph']))
 			$config['ntpd']['statsgraph'] = $_POST['statsgraph'];
 		elseif (isset($config['ntpd']['statsgraph']))
 			unset($config['ntpd']['statsgraph']);
+		if (!empty($enable_rrd_graphing))
```

Comment:
It would be better to check using isset() here to avoid PHP complain when variable is not set